apps/nextjs-website/src/components/molecules/RelatedResources/RelatedResources.tsx

```diff
@@ -0,0 +1,23 @@
+import EContainer from '@/editorialComponents/EContainer/EContainer';
+import { Typography } from '@mui/material';
+import { useTranslations } from 'next-intl';
+import { GuideCard, GuideCardProps } from '../GuideCard/GuideCard';
+
+interface RelatedResourcesProps {
+  guide: GuideCardProps;
+}
```

Comment:
Without the `downloadableDocuments` property this component is not complete.
You should complete DEV-1806 and possibly add a dependency in this PR